examples/platform/asr/ASRUtils.cpp

```diff
@@ -0,0 +1,72 @@
+#include "ASRUtils.h"
+#include "AppConfig.h"
+#include <lega_rtos_api.h>
+
+using namespace ::chip;
+using namespace ::chip::Credentials;
+// using namespace ::chip::DeviceManager;
+using namespace ::chip::DeviceLayer;
+
+extern "C" void asr_matter_reset(int type);
+extern "C" void matter_ble_start_adv(void);
+extern "C" void ShutdownChip();
+
+void asr_matter_reset(int type)
+{
+    if (type == 0)
```

Comment:
why is type an int? Can it be an enum for readability purposes?

Reply:
see matter_shell.h ，type use enum